lib/internal/main/embedding.js

```diff
@@ -98,7 +98,7 @@ let warnedAboutBuiltins = false;
 function embedderRequire(id) {
   const normalizedId = normalizeRequirableId(id);
   if (!normalizedId) {
-    if (isLoadingSea && !warnedAboutBuiltins) {
+    if (isLoadingSea && isExperimentalSeaWarningNeeded && !warnedAboutBuiltins) {
```

Comment:
```suggestion
    if (isLoadingSea && isExperimentalSeaWarningNeeded() && !warnedAboutBuiltins) {
```

Reply:
Or maybe just cache this value above L33?

Reply:
Thanks for the review. I've made the changes.